src/time.cr

```diff
@@ -474,7 +474,7 @@ struct Time
     epoch.to_f + nanosecond.to_f / 1e9
   end
 
-  # Converts this time to UTC.
+  # Returns a copy of this `Time` converted to UTC.
```

Comment:
Not sure it's important, but it's not always a copy: if `self` is already an UTC time, it returns `self`, not a copy of it.

Reply:
As a `struct`, it's still a copy.

Reply:
@oprypin Thanks!

Reply:
oh right I forgot that _minor_ fact